potassium.gemspec

```diff
@@ -18,16 +18,16 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.7"
+  spec.add_development_dependency "bundler", "~> 2.0"
```

Comment:
`~> 2.1` ?

Reply:
for some unknown mystical reason it wont work with bundler >2.0.2